### PR TITLE
Windows subsystem for linux

### PR DESCRIPTION
**Pull request by william-lohan**
add blog

### CODE REVIEW

**Review by crutchcorn — CHANGES_REQUESTED** [review]
OMG, thank you so so much for the contribution! This was so exciting to receive a GitHub email about! :D 

I left a few small comments (the only things I'd want us to change before launching is the `description` and the alt text, but after that, we're good to go!) but overall this is a great article - can't wait to have it go live!

File: content/blog/windows-subsystem-for-linux/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Building dependency tree
+Reading state information... Done
+The following additional packages will be installed:
+...
+Do you want to continue? [Y/n] Y
+...
+user@MACHINE_NAME:~$
+```
+
+_Note: If you are looking to get into C or C++ development `build-essential` is a good package to remember to get compilers_
```
Comment (william-lohan) [inline content/blog/windows-subsystem-for-linux/index.md:97]:
```suggestion
> Note: If you are looking to get into C or C++ development `build-essential` is a good package to remember to get compilers
```

File: content/data/unicorns.json [hunk 2]
```diff
@@ -276,5 +276,22 @@
...
+    	"lastName": "Lohan",
+    	"description": "",
+    	"socials": {
+			"github": "william-lohan",
+   			"twitch": "splat_killwill",
+   			"website": "https://gatimus.com/",
+			"linkedIn": "william-lohan-b202637a"
+    	},
+    	"pronouns": "they/themselves",
+    	"profileImg": "./splatkillwill.jpg",
```
[the remark this answers is not in the document]
Comment (william-lohan) [reply content/data/unicorns.json:293]: thanks :)

File: content/blog/windows-subsystem-for-linux/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+int main()
+{
+  std::cout << "Hello World!\n";
+  return 0;
+}
+```
+
+![](./Screenshot_2021-10-21_205232.jpg)
+
+<!-- Does this need a conclusion paragraph (I suck at those) -->
```
[the remark this answers is not in the document]
Comment (william-lohan) [reply content/blog/windows-subsystem-for-linux/index.md:121]: I forgot about this and it was copy pasted. I think Ill just remove for now.

```suggestion
```

**Pull request merged**